package.json

```diff
@@ -42,7 +42,7 @@
     "@angular/router": "~8.1.2",
     "@cospired/i18n-iso-languages": "^2.0.5",
     "@ngx-translate/core": "^11.0.1",
-    "@rero/ng-core": "0.0.6",
+    "@rero/ng-core": "0.0.7",
```

Comment:
version 0.0.8 of ng-core has been published

Reply:
done